Mage.Sets/src/mage/cards/z/ZurEternalSchemer.java

```diff
@@ -0,0 +1,128 @@
+    }
+
+    public ZurEternalSchemer(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{W}{U}{B}");
+
+        addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.WIZARD);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(4);
+
+        // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // Enchantment creatures you control have deathtouch, lifelink, and hexproof.
+        CompoundAbility compoundAbilities = new CompoundAbility(DeathtouchAbility.getInstance(), LifelinkAbility.getInstance(), HexproofAbility.getInstance());
+        this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new GainAbilityControlledEffect(compoundAbilities, Duration.WhileOnBattlefield, filter)));
+
+        // {1}{W}: Target non-Aura enchantment you control becomes a creature in addition to its other types and has base power and toughness each equal to its mana value.
+        Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, new ZurEternalSchemerEffect(), new ManaCostsImpl<>("{1}{W}"));
+        ability.addTarget(new TargetPermanent(filter2));
+        this.addAbility(ability);
+    }
+
+    private ZurEternalSchemer(final ZurEternalSchemer card) {
+        super (card);
+    }
+
+    @Override
+    public ZurEternalSchemer copy() {
+        return new ZurEternalSchemer(this);
+    }
+}
+
+class ZurEternalSchemerEffect extends ContinuousEffectImpl {
+
+    public ZurEternalSchemerEffect() {
+        super(Duration.EndOfGame, Outcome.BecomeCreature);
+        staticText = "Target non-Aura enchantment you control becomes a creature in addition to its other types " +
+                "and has base power and base toughness each equal to its mana value.";
+    }
+
+    public ZurEternalSchemerEffect(final ZurEternalSchemerEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public ZurEternalSchemerEffect copy() {
+        return new ZurEternalSchemerEffect(this);
+    }
+
+    @Override
+    public boolean apply(Layer layer, SubLayer sublayer, Ability source, Game game) {
+        Permanent enchantment = game.getPermanent(this.getTargetPointer().getFirst(game, source));
+        if (enchantment == null) {
+            return false;
+        }
+        switch (layer) {
+            case TypeChangingEffects_4:
+                if (sublayer == SubLayer.NA) {
+                    if (!enchantment.isCreature(game)) {
+                        enchantment.addCardType(game, CardType.CREATURE);
+                    }
+                }
+                break;
+
+            case PTChangingEffects_7:
+                if (sublayer == SubLayer.SetPT_7b) {
+                    int cmc = enchantment.getManaValue();
+                    enchantment.getPower().setValue(cmc);
+                    enchantment.getToughness().setValue(cmc);
```

Comment:
This part won't work correctly. .setValue sets the final PT rather than the base value. I engine does not currently support this.

I am working on it: https://github.com/magefree/mage/pull/9328#discussion_r947275403

Reply:
Fixed in 31c095a, also fixed cards with the [same effect](https://scryfall.com/search?q=o%3A%22creature+in+addition+to+its+other+types+and+has+base+power+and+base+toughness+each+equal+to+its+mana+value%22&unique=cards&as=grid&order=name)